ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LT:
+  description: "Luminaire, lighting fixture."
+  implements:
+  - EQUIPMENT
+  uses:
+  - dimmer_percentage_command # in % indicating brightness
+
+LPS:
+  description: "Lighting photocell sensor."
+  implements:
+  - MTS	
+
+LMS:
+  description: "Lighting motion sensor."
+  implements:
+  - MTS	
+
+LTGW:
+  description: "Lighting gateway."
+  implements:
+  - EQUIPMENT			
+  
+LAC:
+  description: "lighting area controller."
+  implements:
+  - EQUIPMENT	
+
+LCM:
+  description: "Lighting control module."
+  implements:
+  - EQUIPMENT	
+
+LKP:
+  description: "Lighting keypad."
+  implements:
+  - USER_INTERFACE		
+
+LIGHTING_GROUP:
+  description: "A group containing one or more luminaires."
+
+LIGHTING_SCENE:
```

Comment:
Is this better as metadata? Is it really an object?

Reply:
From discussions we have just now on the UDMI side, we're leaning towards modelling groups and scenes as "virtual" devices, so that it is possible to send configuration events for group of luminaires. The problem arises from the serial nature of lighting communication protocols and the issue that sending individual commands to each controller and luminaire means that synchronisation of lighting events (switch all lights in the building at the same time) cannot work. With the group command, a single command is issued to the full building network and the devices will react accordingly. So yes, we have a use case for modelling the group and scene as "virtual" objects.